File: Python(create_db)/models/product.py
from pydantic import BaseModel
from sqlite3 import Date


class BaseProduct(BaseModel):
    product_type_id: int
    name: str
    price: int
    manufacture_date: Date
    material: str
    color: str
    country: str
    weight: int

    class Config:
        orm_mode = True


class ProductInOrder(BaseProduct):
    id: int
    count: int

    class Config:
        orm_mode = True


class ProductWithWish(BaseProduct):
    id: int
    is_wished: bool = False

    class Config:
        orm_mode = True


class PriceAndCount(BaseModel):
    price: int
    count: int

File: Kotlin(android_app)/app/java/jewelrystore/models/Product.kt
package com.sysoliatina.jewelrystore.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Product(
    val id: Int,
    @SerializedName("product_type_id") val productTypeId: Int,
    val name: String,
    val price: Int,
    @SerializedName("manufacture_date") val manufactureDate: String,
    val material: String,
    val color: String,
    val country: String,
    val weight: Int,
    val count: Int?,
    @SerializedName("is_wished") val isWished: Boolean?
) : Parcelable
